server/src/immich/api-v1/asset/asset.service.ts

```diff
@@ -196,7 +196,7 @@ export class AssetService {
     const includeMetadata = this.getExifPermission(authUser);
     const asset = await this._assetRepository.getById(assetId);
     if (includeMetadata) {
-      const data = mapAsset(asset);
+      const data = mapAsset(asset, false, true);
```

Comment:
This is hard to read imo, we should pass an options object instead.

Reply:
Updated to an options object.